modules/puppet/templates/puppet-darwin-run-puppet.sh.erb

```diff
@@ -94,7 +94,7 @@ try:
    smtpObj = smtplib.SMTP('<%= @smtp_relay_host %>')
    smtpObj.sendmail("${SENDER}", "${RECEIVER}", msg)
    print "Successfully sent email"
-except SMTPException:
+except smtplib.SMTPException:
    print "Error: unable to send email"
```

Comment:
Why catch the exception at all?   You're at the tail end of a short embedded python script that's about to exit (whether it succeeds or fails) and throw control back to a bash script that doesn't seem to be `-e`.  The `except` hides the traceback and is making sending issues undiagnosable.  All this seems to get you is brevity on failures.

Reply:
That's a good point.
If no one objects, I'll remove the try/except.